### PR TITLE
Create Install_PACTA_as_a_first-time_user

### PR DESCRIPTION
**Pull request by meggpie**
The first draft for review, will add the generating reports section ASAP.

### CODE REVIEW

File: documentation/Install_PACTA_as_a_first-time_user [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+### Step 2 — Launch RStudio:
+To launch RStudio with the PACTA_analysis files:
+* From your local directory open` PACTA_analysis > R > PACTA_analysis.rproj `
+
+### Step 3 — Running web_tool_script1 and web_tool_script2:
+The scripts use the following example files located in `working_directory`:
+* `20_Raw_Inputs > TestPortolio_Input.csv`
+* `10_Parameter_File > TestPortfolio_Input_PortfolioParameters.yml`
+	* The `portfolio_name_in` variable must match the `.csv` file name
+
```
Comment (jdhoffa) [inline documentation/Install_PACTA_as_a_first-time_user:51]: Between steps 2 and 3, we are missing an explanation of how to install dependencies. 
@meggpie this did not have a clear solution while we were chatting with you, so not meant to sound as a criticism of this draft, but I think this needs to get solved before we complete this @cjyetman @AlexAxthelm

**Review by jdhoffa — CHANGES_REQUESTED** [review]

File: documentation/Install_PACTA_as_a_first-time_user (hunk 1 above)
Comment (jdhoffa) [reply documentation/Install_PACTA_as_a_first-time_user:51]: Relates to the conversation here: https://github.com/2DegreesInvesting/PACTA_analysis/pull/514#issuecomment-977806196
The issue here: #466
etc.

Comment (jdhoffa) [reply documentation/Install_PACTA_as_a_first-time_user:51]: @cjyetman I just want to follow up on this PR. Regarding my points above, should the Step between Step 2 and 3 be "install all packages listed in `README`?"

Comment (cjyetman) [reply documentation/Install_PACTA_as_a_first-time_user:51]: that seems reasonable to me

Comment (meggpie) [reply documentation/Install_PACTA_as_a_first-time_user]: added that step, just needs instructions on how to install if someone could provide or chat with me sometime next week @jdhoffa @cjyetman @AlexAxthelm

File: documentation/Install_PACTA_as_a_first-time_user [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+**Keywords:**
+
+## Before you start:
+### System requirements:
+|           | Recommended   | Minimum               |
+|-----------|---------------|-----------------------|
+| RAM       | 16 GB or more | 8 GB                  |
+| Windows   | 64-bit        | 32-bit                |
+| Mac OS    |               |                       |
```
Comment (meggpie) [inline documentation/Install_PACTA_as_a_first-time_user:14]: @AlexAxthelm do you have info on system requirements?

File: documentation/Install_PACTA_as_a_first-time_user [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+### Step 2 — Load your portfolio files:
+1. Copy your working portfolio into `20_Raw_Inputs`
+2. Go to `10_Parameter_Files`. Make a copy of `Portfolio_Input_PortfolioParamaters.yml`. Rename the file to `YOURPORTFOLIONAME_PortfolioParameters.yml.`
+3. View the file in RStudio and edit the following variables:
+	* `portfolio_name_in`: your file name without `.csv`
+	* `investor_name_in:` anything
+	* `peergroup:` anything
+	* `language:`available languages?
+	* `project_code:` GENERAL
+	* — placeholder for upcoming feautre— `holdings_date:`
```
Comment (meggpie) [inline documentation/Install_PACTA_as_a_first-time_user:96]: Placeholder for holdings_date feature to be documented, will we include this in this draft?

File: documentation/Install_PACTA_as_a_first-time_user (hunk 2 above)
Comment (AlexAxthelm) [reply documentation/Install_PACTA_as_a_first-time_user:14]: @meggpie same across all platforms.

File: documentation/Install_PACTA_as_a_first-time_user (hunk 3 above)
Comment (AlexAxthelm) [reply documentation/Install_PACTA_as_a_first-time_user:96]: yes please.

`holdings_date: 2020Q4`

**Comment by meggpie** [comment]
@AlexAxthelm did we want to keep working on this?

**Comment by jdhoffa** [comment]
@AlexAxthelm @cjyetman can we merge or close this?

**Comment by jdhoffa** [comment]
hahahaah

**Pull request merged**